activities/Measure.activity/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+<meta charset="utf-8" />
+<title>Measure Activity</title>
+<script type="text/javascript" src="lib/p5.js" ></script>
+  <script type="text/javascript" src="lib/p5.sound.js"></script>
```

Comment:
Use p5.sound.min.js and delete p5.sound.js

Reply:
Fixed